testchain/node/src/service.rs

```diff
@@ -24,6 +25,10 @@ use spacewalk_runtime_testnet::RuntimeApi as TestnetRuntimeApi;
 // Native executor instance.
 pub struct TestnetExecutor;
 
+/// The minimum period of blocks on which justifications will be
+/// imported and generated.
+const GRANDPA_JUSTIFICATION_PERIOD: u32 = 512;
```

Comment:
Have we tried fiddling with this parameter? I found a [test](https://github.com/paritytech/polkadot-sdk/blob/5dc0670a85d575480e0840204c20a5771cd8f7d3/substrate/client/consensus/grandpa/src/tests.rs#L327) in the polkadot-sdk where it's defined to just 32. I honestly don't really understand the implications of this parameter, just something that came to my mind.

Reply:
You mean for the test issue? No, this is the standard value [used](https://github.com/paritytech/polkadot-sdk/blob/dc4047c7d05f593757328394d1accf0eb382d709/templates/solochain/node/src/service.rs#L24) in one of the templates of the `polkadot-sdk`. Yet, for our instant seal this shouldn't matter since we are not using the consensus at all.